docker/docker-compose.yml

```diff
@@ -46,6 +47,13 @@ services:
     ports:
       - '5435:5432'
 
+  cockroachdb:
+    image: prismagraphql/build:cockroach-custom
```

Comment:
Do we have an ARM64 build of this image? If not, it would be nice to add an override to `docker-compose.arm64.yml` with either an alternative image or just

```yaml
cockroachdb:
  profiles:
    - donotstart
```

and skip the test based on some condition.

Reply:
(not blocking this PR because of this, I can do that separately after it lands on `main`)

Reply:
The image works on ARM64. I've been using it locally.